util/uuid.c

```diff
@@ -19,14 +20,15 @@ bool generate_uuid(char out[static 37]) {
 	if (status != uuid_s_ok) {
 		return false;
 	}
-	char *str;
+	char str[37];
+	memset(str, 0, 37);
 	uuid_to_string(&uuid, &str, &status);
```

Comment:
Is the `str` change actually necessary, or just the `37` for `memset` below?

I'm not sure if this is a valid reference, but the man page states that `str` is guaranteed to be of sufficient length (36, apparently).

From https://www.unix.com/man-page/FreeBSD/3/uuid_to_string/:
> The uuid_to_string() function set *str to be a pointer to a buffer sufficiently large to hold the string.	This pointer should be passed to free(3) to release the allocated storage when it is no longer needed.

Reply:
37 should only be necessary for the `memcpy` below. I'll amend.